packages/database/prisma/schema.prisma

```diff
@@ -559,10 +562,6 @@ model Environment {
 model Lock {
   id         String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
   name       String   @unique
-  acquiredAt DateTime @default(now())
-  expiresAt  DateTime
-  isLocked   Boolean  @default(false)
-  ownerId    String   @db.Uuid
```

Comment:
we should make a 2 step process here, lets create another Lock table without these fields